services/src/main/java/org/keycloak/broker/saml/mappers/AttributeToRoleMapper.java

```diff
@@ -81,6 +82,13 @@
 
     public static final String PROVIDER_ID = "saml-role-idp-mapper";
 
+    private static final String emptyToNull(String value) {
```

Comment:
Your StringUtil.isNotNull() and StringUtil.isNullOrEmpty() do and do not trim whitespace, should I make them consistent as well?